automation/project/Dependencies.scala

```diff
@@ -7,10 +7,18 @@ object Dependencies {
 
   val workbenchModelV  = "0.8-d97f551"
   val workbenchModel: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-model" % workbenchModelV
-  val excludeWorkbenchModel = ExclusionRule(organization = "org.broadinstitute.dsde.workbench", name = "workbench-model_2.11")
 
-  val workbenchGoogleV = "0.10-d97f551"
-  val workbenchGoogle: ModuleID = "org.broadinstitute.dsde.workbench" %% "workbench-google" % workbenchGoogleV excludeAll excludeWorkbenchModel
+  val workbenchMetricsV  = "0.3-573431c-SNAP"
```

Comment:
Need to update these after https://github.com/broadinstitute/workbench-libs/pull/55